src/utills/exports-from-folder.ts

```diff
@@ -0,0 +1,11 @@
+import { readdir } from 'fs';
+import { join } from 'path';
+import * as utills from 'util';
+
+const promiseReadDir = utills.promisify(readdir);
+const flatArr = (arr: any[] = []) => [].concat(...arr);
+
+export const getAllInFolder = async (path: string) => {
```

Comment:
Looks like it does not work
you will return just the data from the first file 

This will solve the problem:
`flatArr(filePaths.map((filePath) => Object.values(require(join(path, filePath)))));`